File: BackEnd/src/main/java/com/unosquare/admin_core/back_end/viewModels/dashboard/EventMessageViewModel.java
package com.unosquare.admin_core.back_end.viewModels.dashboard;

import lombok.Data;

@Data
public class EventMessageViewModel {
    private int eventMessageId;
    private String author;
    private String lastModified;
    private String message;
    private int messageTypeId;
    private String messageTypeDescription;
}

File: v2/AdminCore.WebApp/Models/EventMessage/EventMessageViewModel.cs
using System;
using AdminCore.WebApi.Models.Employee;

namespace AdminCore.WebApi.Models.EventMessage
{
  public class EventMessageViewModel
  {
    public int EventMessageId { get; set; }

    public int EventId { get; set; }

    public EmployeeViewModel Employee { get; set; }

    public string Message { get; set; }

    public DateTime LastModified { get; set; }
  }
}